### ISSUE TITLE
Subparser alias is not shown in --help output

### ISSUE THREAD

**Issue by msikma** [nodeca/argparse#85]
JS example script:

``` js
#!/usr/bin/env node

var ArgumentParser = require('argparse').ArgumentParser;

var parser, subparsers, args;
var testArg1, testArg2;

// Main parser setup.
parser = new ArgumentParser({
  'version': '1.0.0',
  'addHelp': true,
  'description': 'Description.',
  'epilog': 'Epilog.'
});
subparsers = parser.addSubparsers({
  dest: 'posarg'
});

// Test arguments.
testArg1 = subparsers.addParser('c1', {addHelp: true, help: 'c1 help'});
testArg1.addArgument(['-f', '--foo'], {
  action: 'store',
  help: 'foo'
});
testArg2 = subparsers.addParser('c2', {
  aliases: ['co'],
  addHelp: true,
  help: 'c2 help'
});
testArg2.addArgument(['-b', '--bar'], {
  action: 'store',
  type: 'int',
  help: 'bar'
});

args = parser.parseArgs();
```

Python example script (running in 3.4):

``` python
#!/usr/bin/env python
import argparse

argparser = argparse.ArgumentParser(add_help=True)
argparser.description = 'Description.'
argparser.epilog = 'Epilog.'
argparser.add_argument('--version', action='version', version='1.0.0')
subparsers = argparser.add_subparsers(
  dest='posarg'
)

test_arg_1 = subparsers.add_parser('c1', help='c1 help')
test_arg_1.add_argument('-f', '--foo', action='store', help='foo')

test_arg_2 = subparsers.add_parser('c2', aliases=['co'], help='c2 help')
test_arg_2.add_argument('-b', '--bar', action='store', type=int, help='bar')

args = argparser.parse_args()
```

When running these scripts, there's one main discrepancy: the JS version shows the second subparser as `c2           c2 help` and the Python version shows `c2 (co)   c2 help`. It would be good to show the alias in the JS version as well.

**Comment by rlidwka** [comment]
Fixed in argparse 2.0, it now displays the following help:

```
usage: test.js [-h] [-v] {c1,c2,co} ...

Description.

positional arguments:
  {c1,c2,co}
    c1           c1 help
    c2 (co)      c2 help

optional arguments:
  -h, --help     show this help message and exit
  -v, --version  show program's version number and exit

Epilog.
```

**Comment by msikma** [comment]
Nice! Forgot I even made this issue...almost 5 years ago.
